src/utils/tests.jl

```diff
@@ -11,13 +11,14 @@ For experimental modules, use [`test_experimental_module`](@ref) instead.
 """
 function test_module(file::AbstractString; new::Bool=true)
   julia_exe = Base.julia_cmd()
+  project_path = split(pathof(Oscar), "/src")[1]
```

Comment:
```suggestion
  project_path = dirname(dirname(pathof(Oscar), "/src")))
```
This should be more robust, e.g. in cases where the path is `/home/myuser/src-code/Oscar.jl/src/Oscar.jl`.

Reply:
Should use `joinpath` and not `/`.

Reply:
How about `Oscar.oscardir`?

Reply:
Maybe even `dirname(Pkg.project().path)`?
The project may have Oscar developed from a different directory.

Reply:
I have used this with good success in various places:
```julia
run(`$(Base.julia_cmd()) --project=$(Base.active_project()) $(ARGS)`)
```